ring-core/src/ring/middleware/multipart_params.clj

```diff
@@ -144,32 +149,47 @@
 
   The following options are accepted
 
-  :encoding          - character encoding to use for multipart parsing.
-                       Overrides the encoding specified in the request. If not
-                       specified, uses the encoding specified in a part named
-                       \"_charset_\", or the content type for each part, or
-                       request character encoding if the part has no encoding,
-                       or \"UTF-8\" if no request character encoding is set.
-
-  :fallback-encoding - specifies the character encoding used in parsing if a
-                       part of the request does not specify encoding in its
-                       content type or no part named \"_charset_\" is present.
-                       Has no effect if :encoding is also set.
-
-  :store             - a function that stores a file upload. The function
-                       should expect a map with :filename, :content-type and
-                       :stream keys, and its return value will be used as the
-                       value for the parameter in the multipart parameter map.
-                       The default storage function is the temp-file-store.
-
-  :progress-fn       - a function that gets called during uploads. The
-                       function should expect four parameters: request,
-                       bytes-read, content-length, and item-count."
+  :encoding                 - character encoding to use for multipart parsing.
```

Comment:
Maybe we should change the formatting so that it's more like:

```
:encoding
- Character encoding...
```

With the addition of `:invalid-filename-handler`, there's just too much whitespace, and the lines go over 80 characters as well.